### PR TITLE
Fix missing log file and apply logging level - Closes #3123

### LINKED ISSUE #3123: Log file is missing and incorrect log level
### Expected behavior
When started with `LISK_NETWORK=testnet` Core writes its log messages to [`logs/testnet/lisk.log`](https://github.com/LiskHQ/lisk/blob/development/config/testnet/config.json#L4).

Logger should use `fileLogLevel` and `consoleLogLevel` parameters passed via `config/{network}/config.json`.

### Actual behavior
The file `logs/testnet/lisk.log` does not get created, no log message are written on disk.

Logger uses default value (`debug`) for both `fileLogLevel` and `consoleLogLevel`.

### Steps to reproduce
`LISK_NETWORK=testnet npm start`

### Which version(s) does this affect? (Environment, OS, etc...)
`1.6`

### PR DESCRIPTION
**Pull request by michielmulders**
### What was the problem?
The file `logs/testnet/lisk.log` does not get created, no log message are written on disk.

### How did I fix it?
Wrong parameter name was used, creating the right folder and file failed because of that.

### How to test it?
Run any network and check if `logs/{network}/lisk.log` gets created.

### Review checklist

* The PR resolves #3123 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

**Comment by michielmulders** [comment]
@fchavant Yes, the bug fix for development, merged it to see if my tests fail/pass. 
Will disappear once merge into development.

**Comment by fchavant** [comment]
The file `config/testnet/config.json` (with `LISK_NETWORK=testnet`) is now present however it contains `debug` level messages while `config/default/config.json` contains `"fileLogLevel": "info",` and `config/testnet/config.json` contains no `fileLogLevel` key; is this a separate bug?

File: framework/src/modules/chain/helpers/config.js [hunk 1]
```diff
@@ -223,11 +214,9 @@ const getenv = (variable, defaultValue = null, isBoolean = false) => {
 
 function loadJSONFile(filePath) {
 	try {
-		filePath = path.resolve(rootPath, filePath);
+		filePath = path.join(rootPath, filePath);
 		return JSON.parse(fs.readFileSync(filePath, 'utf8'));
 	} catch (err) {
-		console.error(`Failed to load file: ${filePath}`);
-		console.error(err.message);
```
[the remark this answers is not in the document]
Comment (michielmulders) [reply framework/src/modules/chain/helpers/config.js]: Good catch, to greedy removing console.log messages

**Comment by michielmulders** [comment]
@fchavant addressed the fix in this issue as well
Will edit the title and issue.
Please test from your side.

File: framework/src/modules/chain/helpers/config.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ const configSchema = require('../schema/config');
 const { ZSchema } = require('../../../controller/helpers/validator');
 
 const validator = new ZSchema();
-const rootPath = path.dirname(path.resolve(__filename, '..'));
+const rootPath = path.dirname(path.resolve(__filename, '../../../../..'));
```
Comment (yatki) [inline framework/src/modules/chain/helpers/config.js]: since we already have root path, I prefer to use in the requires below, to make it more clear.

**Pull request merged**